### ISSUE TITLE
Support starting stopped EC2 instances by tag

### ISSUE THREAD

**Issue by lorin** [ansible/ansible-modules-core#138]
##### Issue Type:

Feature Idea
##### Ansible Version:

ansible 1.7.2
##### Environment:

N/A
##### Summary:

Allow user to start all stopped EC2 instances associated with a particular tag.
##### Steps To Reproduce:

Extend the ec2 module to enable, say, starting all stopped instances tagged with key=Name, Value=ExtraPower

```
   - local_action:
      module: ec2
      instance_tags:
            Name: ExtraPower
      state: running

```
##### Expected Results:

Any stopped instance with the associated tag will be started (any instance already running with the tag would be unaffected).
##### Actual Results:

N/A

**Comment by lorin** [comment]
I implemented this here: https://github.com/ansible/ansible/pull/7197, need to port it over to the new code organization.

**Comment by zoza** [comment]
Hi, any chance that this code will be merged to upstream??

**Comment by juliedavila** [comment]
@zoza, @lorin could you make a new pull request to ansible-modules-core from your original p/r?

**Comment by lorin** [comment]
@defionscode looks like @wtcross ported it: https://github.com/ansible/ansible-modules-core/pull/1175

**Comment by juliedavila** [comment]
@gregdek @bcoca this one can be closed, function merged in here: https://github.com/ansible/ansible-modules-core/pull/1703

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
